nncf/experimental/openvino_native/quantization/algorithms/fast_bias_correction/ov_backend.py

```diff
@@ -0,0 +1,167 @@
+        return OVModelExtractionCommand(inputs, outputs)
+
+    @staticmethod
+    def mean_statistic_collector(reduction_shape:
+                                 ReductionShape,
+                                 num_samples: int = None,
+                                 window_size: int = None) -> OVMeanStatisticCollector:
+        return OVMeanStatisticCollector(reduction_shape, num_samples, window_size)
+
+    @staticmethod
+    def get_input_output_names(node: NNCFNode, nncf_graph: NNCFGraph) -> Tuple[str, str]:
+        input_name = node.node_name
+        output_name = node.node_name
+
+        bias_node = OVFastBiasCorrectionAlgoBackend.get_bias_node(node, nncf_graph)
+        if bias_node is not None:
+            output_name = bias_node.node_name
+        return input_name, output_name
+
+    @staticmethod
+    def create_blob(shape: Tuple[int], data: List[float]) -> np.ndarray:
+        blob = np.zeros(shape)
+        for i, value in enumerate(data):
+            blob[:, i] = value
+        blob = blob.astype(np.float32)
+        return blob
+
+    @staticmethod
+    def get_bias_value(model: ov.Model, bias_node: NNCFNode) -> np.ndarray:
+        ops_dict = {op.get_friendly_name(): op for op in model.get_ops()}
+
+        bias_node = ops_dict[bias_node.node_name]
+        skip_metatypes = ['Convert']
+        needed_bias_type = 'Constant'
+
+        potential_bias_queue = deque([n.node for n in bias_node.input_values()])
+        while potential_bias_queue:
+            potential_bias = potential_bias_queue.popleft()
+            if potential_bias.get_type_name() in skip_metatypes:
+                # We goes thorough 0 port in assumption that bias graph without branching
+                potential_bias_queue.append(potential_bias.input_value(0).node)
+            elif potential_bias.get_type_name() == needed_bias_type:
+                bias_value = potential_bias.get_vector()
+                return bias_value.flatten(), bias_value.shape
+            continue
+
+        raise RuntimeError('Could not find the bias value of the node')
+
+    @staticmethod
+    def get_activation_port_ids_for_bias_node(model: ov.Model, biased_node: NNCFNode) -> Tuple[int, int]:
+        return 0, 0
+
+    @staticmethod
+    def is_quantized_weights(biased_node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        weight_port_id = biased_node.layer_attributes.weight_port_id
+        weight_node = nncf_graph.get_input_edges(biased_node)[weight_port_id].from_node
+        return weight_node.metatype == OVFakeQuantizeMetatype
+
+    @staticmethod
+    def get_bias_port_id(bias_node: NNCFNode) -> int:
+        return bias_node.layer_attributes.weight_port_id
+
+    @staticmethod
+    def process_model_output(raw_data: Dict, output_name: str) -> OVNNCFTensor:
+        return OVNNCFTensor(raw_data[output_name])
+
+    @staticmethod
+    def is_node_with_bias(biased_node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
```

Comment:
Could you please move this method's body to the `nncf/experimental/openvino_native/graph/node_utils.py` module as a separate method?